docs/how-to-guide/how-to-guide.md

```diff
@@ -0,0 +1,358 @@
+# How-to guide
+
+### Configure WordPress\*
+
+\*This only works when setting up WordPress initially. Changing the value afterwards has no effect.
```

Comment:
This doesn't stand out much when previewing:(

Reply:
I meant to add an asterisk as a side note, like "buy one get one free*"
"*terms and conditions may apply" kind of a footnote. Would this require additional highlighting? :)

Reply:
I guess it doesn't